app/helpers/lookup_keys_helper.rb

```diff
@@ -118,8 +118,8 @@ def lookup_key_description(lookup_key, matcher, inherited_value)
      <b>Type:</b> %{type}<br/>
      <b>Matcher:</b> %{matcher}<br/>
      <b>Inherited value:</b> %{inherited_value}") %
-    { :desc => lookup_key.description, :type => lookup_key.key_type,
-      :matcher => matcher, :inherited_value => inherited_value }
+    { :desc => html_escape(lookup_key.description), :type => lookup_key.key_type,
+      :matcher => matcher, :inherited_value => html_escape(inherited_value) }
```

Comment:
Could a matcher also contain HTML content?  There's not a lot of validation on them, I was thinking of something like `os=<script>...`